pyiron_base/jobs/job/core.py

```diff
@@ -429,6 +431,50 @@ def remove(self, _protect_childs=True):
         # After all children are deleted, remove the job itself.
         self.remove_child()
 
+    def kill(self, enable=True):
```

Comment:
```suggestion
    def kill(self, enable=False):
```
Because the I guess the point was to avoid people accidentally calling it.

Reply:
See above - the enable would be a "force" option to attempt the job kill with no safeties regarding job status. I'll switch it to False by default (statuses are checked before job kill is attempted), as you suggest, but the `enable` flag here is not to protect the user, but to safeguard against killing jobs which statuses are not in the list.